docs/03-github/04-builder.mdx

```diff
@@ -454,6 +454,30 @@ to avoid any conflicts.
 
 _**required:** `false`_ _**default:** `"/github/workspace"`_
 
+#### dockerCpuLimit
+
+Number of CPU cores to assign the Windows docker container. Defaults to all available cores when no
+value is specified. Can accept fractional values, ie 0.5 for half of a cpu core's execution time.
+
+_**required:** `false`_ _**default:** `""`_
+
+#### dockerMemoryLimit
+
+Amount of memory to assign the docker container. Defaults to 95% of total system memory rounded down
```

Comment:
nitpick: simply wondering if we should have consistency with "Windows docker container" vs "docker container" in here.